src/components/SimpleCard/markup.tsx

```diff
@@ -7,39 +7,50 @@ import { ShopifyImage, ShopifyMoney, ShopifyProduct, ShopifyVariant } from "@/sh
 import { generateCarouselHTML } from "./carousel"
 import { parseId } from "@/shopify/graphql/utils"
 
-export function generateCardHTML(element: SimpleCard, product: ShopifyProduct) {
+export function generateCardHTML(element: SimpleCard, product: ShopifyProduct): HTMLElement {
   const hasDiscount = element.discount && isDiscounted(product)
 
   const prices = (element.variantId && product.variants.find(v => parseId(v.id) === element.variantId)) || product
 
-  return html`
+  return (
```

Comment:
turn all HTMLElement emitting functions used in this jsx expression into JSX component functions